app/controllers/provider_foreman_controller.rb

```diff
@@ -36,12 +36,12 @@ def show_list
   def new
     assert_privileges("provider_foreman_add_provider")
     @provider_cfgmgmt = ManageIQ::Providers::ConfigurationManager.new
-    @provider_types = ["Ansible Tower", "Foreman"]
+    @provider_types = ["Ansible Tower", ui_lookup(:ui_title => 'foreman')]
```

Comment:
Left something out 😄 

So, once you have all the providers, then there should be a description on each one, which is where this PR comes into play...that is, the UI should just show all the descriptions of all the providers it finds via lookup.  Not sure who is responsible for i18n of that (if you even need to i18n a provider name), but at least the Satellite trick for Foreman can be handled at the model level.